.github/workflows/build-checking.yml

```diff
@@ -0,0 +1,42 @@
+name: Check Build
+
+env:
+  DRAGONFLY_PATH: "${{github.workspace}}/build-opt/dragonfly"
+  
+on: 
+  schedule:
+    - cron: '0 7 */3 * *'
+  workflow_dispatch:
+      
+jobs:
+  build-n-test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+        with:
+          submodules: true
+      - name: Creating build
+        run: |
+          sudo apt-get update
```

Comment:
you can do this, or alternatively you can use the docker dev image "ubuntu-dev:20" to save few minutes per run.